projects/storefrontstyles/scss/_components.scss

```diff
@@ -31,14 +32,15 @@ $selectors: mergeAll(
     $myaccount-components-whitelist,
     $store-finder-components-whitelist,
     $asm-components-whitelist,
-    $pwa-components-whitelist
+    $pwa-components-whitelist,
+    $cds-components-whitelist
   )
 );
 
 @each $selector in $selectors {
   #{$selector} {
     // skip selectors if they're added to the $skipComponentStyles list
-    @if (index($skipComponentStyles, $selector) == null) {
```

Comment:
please revert this change

Reply:
It looks like this was reverted in one of the later commits, as I see this line in the feature/COL-168-cms-compent branch https://github.com/SAP/cloud-commerce-spartacus-storefront/blob/feature/COL-168-cms-component/projects/storefrontstyles/scss/_components.scss

Reply:
Develop has it like this https://github.com/SAP/cloud-commerce-spartacus-storefront/blob/develop/projects/storefrontstyles/scss/_components.scss

Reply:
Its also like this in this feature branch https://github.com/SAP/cloud-commerce-spartacus-storefront/blob/feature/COL-168-cms-component/projects/storefrontstyles/scss/_components.scss, or am I missing something?

Reply:
that link shows me this 
<img width="1058" alt="Screen Shot 2019-11-13 at 5 19 00 PM" src="https://user-images.githubusercontent.com/29099466/68809371-ca96bb80-0639-11ea-9c42-cc0cae98cce7.png">

They should have a space, and prettier was supposed to fix it

Reply:
This is a specific commit and has been resolved in a later commit

Reply:
Hmm, I've fixed it manually, but prettier didn't correct it, not did the ./build.sh sonar file. Its fixed now